week1/assignment/todo-loi/list.rb

```diff
@@ -0,0 +1,64 @@
+require 'colorize'
+
+class List
+  attr_accessor :items
+
+  def initialize(items)
+    @items = items
+  end
+
+  def seperate
+    puts ("-" * 75).colorize(:green)        
+  end
+
+  def undone_items
+    temp = []
+    @items.each do |item|
+      if item.undone?
+        temp << item
+      end
+    end
+    temp
+  end
+
+  def all
+    puts "Todo List:".colorize(:green)
+    puts ("-" * 75).colorize(:green)
+    @items.each_with_index { |e, idx| puts (idx+1).to_s + ". " + e.line + "\n"}
+  end
+
+  def add
+    print "What would you like to add? ".colorize(:green)
+    item = gets.chomp
+    puts "\nOkay, adding ".colorize(:green) + "'#{item}'" + " to the Todo list.\n".colorize(:green)
+    new_item = Item.new("[ ] " + item)
+    @items << new_item
+    all
+  end
+
+  def remove
+    print "Which item would you like to" + " remove? ".colorize(:red)
+    input = gets.chomp    
+    puts "\nOk, removing item number ".colorize(:green) + "'#{input}'" + " from your list.\n".colorize(:green)
+    @items.delete_at(input.to_i - 1)
```

Comment:
you may want to prevent deleting the last time `-1` if the user enters invalid input such as `asdf` (which results to 0 index after `.to_i`)